### PR TITLE
Pull request review for GLM-16 -> sprint_2

### PR DESCRIPTION
**Pull request by hedvik**

### CODE REVIEW

File: api_layer/source/models/Match.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const MatchSchema = mongoose.Schema({
 MatchSchema.statics.findByToken = function (gameToken) {
     return new Promise((resolve, reject) => {
         this.find({ gameToken }).exec()
-            .then(match => resolve(match))
+            .then(matches => resolve(matches))
```
Comment (hedvik) [inline api_layer/source/models/Match.js:25]: The database collection should be called matches and not match

File: api_layer/source/controllers/Matches.js [hunk 2]
```diff
@@ -7,4 +7,23 @@ module.exports = (api) => {
...
+     * @apiGroup Matches
+     * @apiDescription
+     * Returns a list of all matches that contain the provided gameToken.
+     * This list is empty if no matches were found.
+     *
+     * @apiParam {String} gameToken The unique game token of a game.
+     * @apiSuccess (200) Success
+     *
+     */
+    api.route('/Matches/:gameToken')
```
Comment (Per-Morten) [inline api_layer/source/controllers/Matches.js]: By convention, we are apprentantly have routes as lowercase. i.e /matches/:gameToken

File: api_layer/source/tests/matches/api.test.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                }
+            }).catch(err => console.log(err))
+    });
+});
+
+const server = require('../../classes/App');
+
+test.serial('Returning matches with given gameToken: test', async(t) => {
+    await request(server)
+        .get('/Matches/' + t.context.matches[0].gameToken)
```
Comment (Per-Morten) [inline api_layer/source/tests/matches/api.test.js]: Also need to update this to "/matches/"

**Review by Per-Morten — CHANGES_REQUESTED** [review]
Minor comment, otherwise seems fine :)

File: api_layer/source/controllers/Matches.js [hunk 4]
```diff
@@ -7,4 +7,23 @@ module.exports = (api) => {
                 .then(list => res.json(list))
                 .catch(err => res.json(err));
         });
+
+    /**
+     * @api {get} /match/:gameToken Request matches with given gameToken
```
Comment (Per-Morten) [inline api_layer/source/controllers/Matches.js:12]: /matches/

**Review by Per-Morten — APPROVED** [review]
Il fix this during merge.

**Pull request merged**